packages/node/src/indexer/dictionary/v2/types.ts

```diff
@@ -78,7 +78,7 @@ export interface EthDictionaryLogConditions {
 }
 
 export interface EthDictionaryTxConditions {
-  to?: string[];
+  to?: (string | null)[];
```

Comment:
I don't think this is right. The implementation is probably incorrect if null can be part of the array

Reply:
There are currently many checks for to being equal to null in the code.